eZ/Bundle/EzPublishRestBundle/Tests/Functional/TestCase.php

```diff
@@ -78,17 +78,30 @@ protected function assertHttpResponseCodeEquals(HttpResponse $response, $expecte
             $errorMessageString = '';
             if (strpos($response->getHeader('Content-Type'), 'application/vnd.ez.api.ErrorMessage+xml') !== false) {
                 $body = \simplexml_load_string($response->getContent());
-                $errorMessageString = $body->errorDescription;
+                $errorMessageString = $this->getHttpResponseCodeErrrorMessage($body);
             } elseif (strpos($response->getHeader('Content-Type'), 'application/vnd.ez.api.ErrorMessage+json') !== false) {
                 $body = json_decode($response->getContent());
-                $errorMessageString = "Error message: {$body->ErrorMessage->errorDescription}\n" .
-                    "In {$body->ErrorMessage->file}:{$body->ErrorMessage->line}";
+                $errorMessageString = $this->getHttpResponseCodeErrrorMessage($body->ErrorMessage);
```

Comment:
Nitpick:
this is passing `$body->ErrorMessage`, but the function expects a `$body` parameter, which looks somewhat confusing at first. perhaps the function parameter could be doc'd / renamed?